### PR TITLE
NMS-13225 Update help page with Antora, OpenAPI, and Swagger UI links

### PR DESCRIPTION
**Pull request by jadyhou**
Update help page with Antora, OpenAPI, and Swagger UI links.

Because we don't have offline doc anymore, the corresponding code is removed here too. 

Adding Bonnie to see if she has suggestions for the wording of the newly added "OpenAPI" and "Swagger UI" description. 

Below is the screenshot of the help page.
 
image

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13225

### CODE REVIEW

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 1]
```diff
@@ -55,21 +55,21 @@
             <table class="table">
                 <tr>
                     <td style="border-top: none;"><a
-                            href="https://docs.opennms.org/opennms/releases/<%=Vault.getProperty("version.display")%>/guide-install/guide-install.html"
+                            href="https://docs.opennms.com/horizon/<%=Vault.getProperty("version.display")%>/deployment/core/introduction.html"
                             target="_blank" class="btn btn-secondary" role="button"
                             style="width: 100%">Installation Guide</a></td>
                     <td style="border-top: none;">OpenNMS can be installed several operating systems and can be deployed for several scenarios with different technologies. Have a look in the Installation Guide to find instructions to deploy and maintain your OpenNMS instance.</td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp:43]:
```suggestion
                    <td style="border-top: none;">Learn how to deploy and configure the OpenNMS core, Minion, and Sentinel. The guide also covers how to set up a message broker, time series storage options,and setting up Jaeger tracing.</td>
```

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 2]
```diff
@@ -55,21 +55,21 @@
...
                 </tr>
                 <tr>
                     <td style="border-top: none;"><a
-                            href="https://docs.opennms.org/opennms/releases/<%=Vault.getProperty("version.display")%>/guide-admin/guide-admin.html"
+                            href="https://docs.opennms.com/horizon/<%=Vault.getProperty("version.display")%>/operation/overview/overview.html#overview"
                             target="_blank" class="btn btn-secondary" role="button"
                             style="width: 100%">Admin Guide</a></td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp:49]:
```suggestion
                            style="width: 100%">Operation Guide</a></td>
```

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 3]
```diff
@@ -55,21 +55,21 @@
...
                     <td style="border-top: none;">Have a look into the Admin Guide to find instructions how to configure OpenNMS to monitor your infrastructure and services.</td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp:50]:
```suggestion
                    <td style="border-top: none;">Learn how to configure OpenNMS to monitor your infrastructure and services.</td>
```

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 4]
```diff
@@ -55,21 +55,21 @@
...
                 </tr>
                 <tr>
                     <td style="border-top: none;"><a
-                            href="https://docs.opennms.org/opennms/releases/<%=Vault.getProperty("version.display")%>/guide-development/guide-development.html"
+                            href="https://docs.opennms.com/horizon/<%=Vault.getProperty("version.display")%>/development/overview/overview.html#overview"
                             target="_blank" class="btn btn-secondary" role="button"
                             style="width: 100%">Developers Guide</a></td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp:56]:
```suggestion
                            style="width: 100%">Development Guide</a></td>
```

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 5]
```diff
@@ -55,21 +55,21 @@
...
                     <td style="border-top: none;">Developers can extend and improve the OpenNMS platform. The Developers Guide is a good starting point for extending OpenNMS and using the ReST APIs for integration.</td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp:57]:
```suggestion
                    <td style="border-top: none;">For developers who want to extend and improve the OpenNMS platform and use the ReST APIs for integration. Learn how to set up a development environment and get started with writing features for OpenNMS.</td>
```

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 6]
```diff
@@ -85,6 +85,18 @@
                             class="btn btn-secondary" role="button" style="width: 100%">Welcome Guide</a></td>
                     <td style="border-top: none;">If you are new in the project, you can find useful information in your Welcome Guide to get anything you need to get started.</td>
                 </tr>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp:69]: The Welcome Guide section (lines 85-87) should be removed, as the link doesn't work, and I don't think we have a welcome guide anymore.

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 7]
```diff
@@ -85,6 +85,18 @@
...
+                <tr>
+                    <td style="border-top: none;"><a
+                            href="api/v2/openapi.json" target="_blank"
+                            class="btn btn-secondary" role="button" style="width: 100%">OpenAPI doc</a></td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp:73]:
```suggestion
                            class="btn btn-secondary" role="button" style="width: 100%">OpenAPI Docs</a></td>
```

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 8]
```diff
@@ -85,6 +85,18 @@
...
+                    <td style="border-top: none;">With OpenAPI doc, you can easily know how each OpenNMS RESTful API works. You can also test and generate client code from it.</td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp]:
```suggestion
                    <td style="border-top: none;"> The OpenAPI docs explains how each OpenNMS RESTful API works. You can also test and generate client code from them.</td>
```

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 9]
```diff
@@ -85,6 +85,18 @@
...
+                </tr>
+                <tr>
+                    <td style="border-top: none;"><a
+                            href="api/v2/api-docs/?url=/opennms/api/v2/openapi.json#/" target="_blank"
+                            class="btn btn-secondary" role="button" style="width: 100%">Swagger UI</a></td>
+                    <td style="border-top: none;">A graph interface of OpenNMS OpenAPI doc.</td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp]:
```suggestion
                    <td style="border-top: none;">A graphical user interface of the OpenNMS OpenAPI docs.</td>
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
@jadyhou It would be good to update the other text on this screen. It it is an issue to update as part of this PR, let me know and I can create a separate ticket. I'd like to update the content on the rest of this screen too.

**Comment by jadyhou** [comment]
> @jadyhou It would be good to update the other text on this screen. If it is an issue to update as part of this PR, let me know and I can create a separate ticket. I'd like to update the content on the rest of this screen too (i.e., the "Get in Contact" section and the "Software Management" section).

Hi Bonnie, I think it's okay to update other wording or outdated links on this page in this pull request. I can update the Jira say there are also other things we also updated.

**Comment by Bonrob2** [comment]
Thanks for updating this screen. It would be great to update the other sections as well:

Change "Get in Contact" to "Contact Us". 

Change "Web Chat" description text to: "Chat with us on Mattermost. The Discussion channel is a good place to reach users and developers."

Change "Discourse" description text to: "Community forum to get help, share ideas, and participate in OpenNMS development."

Under Software Management:

Change "Issue Tracker" description text to: "We use Jira to manage the software project: bugs, enhancements, features, and release management. Create your free account. You are welcome to participate."

Change "Continuous Integration" button link to the following: https://app.circleci.com/pipelines/github/OpenNMS/opennms

And the description text should be: "Running OpenNMS in production requires software quality gates provided by CircleCI, our CI/CD system. GitHub login required."

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 10]
```diff
@@ -39,45 +39,39 @@
...
-                            class="btn btn-secondary" role="button" style="width: 100%">Welcome Guide</a></td>
-                    <td style="border-top: none;">If you are new in the project, you can find useful information in your Welcome Guide to get anything you need to get started.</td>
-                </tr>
                 <tr>
                     <td style="border-top: none;"><a
                             href="api/v2/openapi.json" target="_blank"
-                            class="btn btn-secondary" role="button" style="width: 100%">OpenAPI doc</a></td>
-                    <td style="border-top: none;">With OpenAPI doc, you can know how each OpenNMS RESTful API works. You can also test the APIs and generate client code from the doc.</td>
+                            class="btn btn-secondary" role="button" style="width: 100%">OpenAPI Docs</a></td>
+                    <td style="border-top: none;">The OpenAPI docs explains how each OpenNMS RESTful API works. You can also test and generate client code from them.</td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp:68]: My bad, I missed removing the "s" on "explains".
```suggestion
                    <td style="border-top: none;">The OpenAPI docs explain how each OpenNMS RESTful API works. You can also test and generate client code from them.</td>
```

File: opennms-webapp/src/main/webapp/includes/help-documentation.jsp [hunk 11]
```diff
@@ -39,45 +39,39 @@
...
                 </tr>
                 <tr>
                     <td style="border-top: none;"><a
                             href="api/v2/api-docs/?url=/opennms/api/v2/openapi.json#/" target="_blank"
                             class="btn btn-secondary" role="button" style="width: 100%">Swagger UI</a></td>
-                    <td style="border-top: none;">This is a graph interface of OpenNMS OpenAPI doc.</td>
+                    <td style="border-top: none;">A graphical user interface of the OpenNMS OpenAPI docs.</td>
```
Comment (Bonrob2) [inline opennms-webapp/src/main/webapp/includes/help-documentation.jsp:74]:
```suggestion
                    <td style="border-top: none;">A graphical user interface for the OpenNMS OpenAPI docs.</td>
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Just two minor issues I had missed.

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Review by cgorantla — APPROVED** [review]

**Comment by jadyhou** [comment]
> This should be `foundation-2021`, shouldn't it? So it makes it into Meridian 2021.

Thanks. I've created a pull request for backporting the Antora part to foundation-2021. Leave the OpenAPI and Swagger UI part in Develop.

**Pull request merged**